splitio/storage/pluggable.py

```diff
@@ -191,5 +204,8 @@ def put(self, split):
         :param split: Split object.
         :type split: splitio.models.split.Split
         """
+        existing_split = self.get(split.name)
+        if existing_split is not None:
+            self._decrease_traffic_type_count(existing_split.traffic_type_name)
         self._pluggable_adapter.set(self._prefix + split.name, split.to_json())
-        self._pluggable_adapter.increment(self._traffic_type_prefix + split.traffic_type_name, 1)
+        self._increase_traffic_type_count(split.traffic_type_name)
```

Comment:
I would call `_increase_traffic_type_count` first, and then `_decrease_traffic_type_count`, to avoid unnecessary wrapper `delete` operations when calling `_decrease_traffic_type_count`

Reply:
will fix